File: template/app/src/main/java/com/xmartlabs/template/ui/login/LoginActivity.java
package com.xmartlabs.template.ui.login;

import android.support.annotation.NonNull;

import com.f2prateek.dart.HensonNavigable;
import com.xmartlabs.bigbang.ui.BaseFragment;
import com.xmartlabs.bigbang.ui.SingleFragmentActivity;

@HensonNavigable
public class LoginActivity extends SingleFragmentActivity {
  @NonNull
  @Override
  protected BaseFragment createFragment() {
    return new LoginFragmentBuilder().build();
  }
}

File: template/app/src/main/java/com/xmartlabs/template/ui/login/LoginActivity.kt
package com.xmartlabs.template.ui.login

import com.f2prateek.dart.HensonNavigable
import com.xmartlabs.bigbang.ui.SingleFragmentActivity

@HensonNavigable
class LoginActivity : SingleFragmentActivity() {
  override fun createFragment() = LoginFragmentBuilder().build()
}
